### ISSUE TITLE
Add unique_junctions = True to Skeleton initialization

### ISSUE THREAD

**Issue by TaekedeHaan** [jni/skan#83]
Thank you very much for this library, it is very useful! While using it I came across the following (minor) problem.

Upon creating a Skeleton object the `skeleton_to_csgraph()` function is called. I would like to to be able to pass  `unique_junctions = False` to this function, which is currently not possible via `__init__()`. As far as my knowledge goes this could easily be added, without breaking anything, by changing `csr.py` as follows:

```
    def __init__(self, skeleton_image, *, spacing=1, source_image=None,
                 _buffer_size_offset=None, keep_images=True, unique_junctions= True):

        graph, coords, degrees = skeleton_to_csgraph(skeleton_image,
                                                     spacing = spacing, 
                                                     unique_junctions = unique_junctions
)
```

Do you think this would be a useful change? Or am I overlooking something?

**Comment by jni** [comment]
@TaekedeHaan this would be very useful! Would you be able to submit a PR with this change?

**Comment by jni** [comment]
Closed by #84
